app-config.yaml

```diff
@@ -464,3 +464,6 @@ apacheAirflow:
 
 gocd:
   baseUrl: https://your.gocd.instance.com
+
+airbrake:
+  apiKey: ${AIRBRAKE_API_KEY}
```

Comment:
Ah, realized one more thing, which is that this is required right? As in if `AIRBRAKE_API_KEY` isn't set the backend will refuse to start up, even in development?

Would like that to not be the case, you could for example make it a soft error in development like some other plugins do. Or take the route that for example `azureDevOps.token` does in this config and just use a mock value

Reply:
Not a problem, I'll take a look and get this sorted in another branch.